### PR TITLE
Feature/et 24 java doc

### PR DESCRIPTION
**Pull request by JakeBat**
## Explanation
Adding javadoc to project as it's required for a maven central release.

### CODE REVIEW

**Comment by ciaran16** [comment]
Are we going for complete docs coverage? I still get some warnings building this branch

I noticed you have some extra docs changes in #9 (adding description to the json files) - is that the fix?

**Comment by JakeBat** [comment]
@ciaran16 Yes, if you build with the changes in https://github.com/onfido/onfido-java/pull/9, which include these changes, there are no longer any warnings, I can pull them into this branch if you'd like.

File: onfido-java/src/main/java/com/onfido/Onfido.java [hunk 1]
```diff
@@ -41,32 +74,68 @@ private Onfido(Builder builder) {
...
+     * @param apiToken the api token
+     * @return the builder
+     */
     public Builder apiToken(String apiToken) {
       this.apiToken = apiToken;
       return this;
     }
 
+    /**
+     * Sets the object to use the US region API.
```
Comment (ben-superdevx) [inline onfido-java/src/main/java/com/onfido/Onfido.java]: I would prefer to say "US region base URL" if that's OK.

File: onfido-java/src/main/java/com/onfido/api/WebhookPayload.java [hunk 2]
```diff
@@ -2,14 +2,27 @@
...
+     * Instantiates a new Webhook payload.
+     *
+     * @param payload the payload
+     */
     protected WebhookPayload(WebhookEvent payload) {
         this.payload = payload;
     }
 
+    /**
+     * Gets payload.
```
Comment (ben-superdevx) [inline onfido-java/src/main/java/com/onfido/api/WebhookPayload.java]: Minor: Could say "Gets Webhook payload"?

**Pull request merged**